dynamic_dynamodb/__init__.py

```diff
@@ -101,6 +104,126 @@ def main():
     except Exception as error:
         logger.exception(error)
 
+def create_time_series_tables(rotate_suffix, rotate_window_start, rotate_scavenge, table_name, table_key, tables_and_gsis, next_table_names):
```

Comment:
There are really only 2 or 3 lines of common functionality shared between the 2 functions. all other lines are different. I dont think merging the 2 makes sense.